lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json

```diff
@@ -2462,7 +2462,7 @@
       "chiller_capacity_guess": null,
       "pszac_cooling_type": "Single Speed Heat Pump",
       "pszac_heating_type": "Single Speed Heat Pump",
-      "pszac_fan_type": "Cycling",
+      "pszac_fan_type": "ConstantVolume",
```

Comment:
@dmaddoxwhite - Is that in line with the unitary system changes you made?

Reply:
I think this should work, and it is more consistent with the DOE small office prototype, which does not have cycling fans.